test/modm/math/saturation/saturation_test.hpp

```diff
@@ -19,14 +19,17 @@ class SaturationTest : public unittest::TestSuite
 {
 public:
 	void
-	testSigned8bit();
+	test__int8_t();
```

Comment:
```suggestion
	test_int8_t();
```
Or is there a reason for the dual underscore?